src/foam/nanos/crunch/AssociationCapability.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Don't think this subclass is necessary. Actually complicates this a bit more than necessary.

Consider:
1) 
CODE:
var isAssociationCapability = self.AssociationCapability.isInstance(self.data);
var associatedEntity = isAssociationCapability ? self.subject.realUser : 
            self.data.associatedEntity === 'user' ? self.subject.user : self.subject.realUser;

above code used in multiple locations (CapabilityCardView, CapabilityIntercept and CrunchController)
If you got rid of this subclass then the code you need becomes:
var associatedEntity = self.subject[self.data.associatedEntity];

We can set the associatedEntity on the capability jrl entry.

2) Also this affects the CrunchController.launchWizard. Now it takes in a capability instead of an id. I think this now adds a layer of complication to the call - instead of just needing the id of the capability you want to call, you'll have to query the ucj for the capability before calling the launchWizard ... seems messy if this is going to be called through api(DIG).

Reply:
using self.subject[self.data.associatedEntity] covers the first 2 cases in my description above but not the third one for "capability that belong to the real user when acting as user"
as for crunchcontroller.launchwizard, I think even without this, the capability needs to be passed in to get its associatedEntity.

Reply:
hmmm I see it now - so AssociationCapability is marker for the AgentCapabilityJunction.
Thx for clarifying.

Reply:
as for the wizard - how about a find in crunchController.launchWizard(id) that first searches the for the capability id - then don't need to change the calls or do a find on each call to the launchWizard